bin/bookkeeper-cli

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
```

Comment:
to un this shell script, do I have to config / set anything, I'm getting following error

/Users/cguttapalem/Workspace/Community/bookkeeper/bin
cguttapalem@cguttapale-ltm1:~/Workspace/Community/bookkeeper/bin$ ./bookkeeper-cli 
JAVA_HOME not set, using java from PATH. (/usr/bin/java)
Fail to load sub command 'cluster' : null
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.apache.bookkeeper.tools.cli.BookieShell.newCommandInstance(BookieShell.java:69)
	at org.apache.bookkeeper.tools.cli.BookieShell.setupShell(BookieShell.java:122)
	at org.apache.bookkeeper.tools.cli.BookieShell.run(BookieShell.java:142)
	at org.apache.bookkeeper.tools.cli.BookieShell.main(BookieShell.java:205)
Caused by: java.lang.NoClassDefFoundError: org/apache/bookkeeper/stats/StatsLogger
	at org.apache.bookkeeper.tools.cli.commands.CmdCluster.<init>(CmdCluster.java:32)
	... 8 more
Caused by: java.lang.ClassNotFoundException: org.apache.bookkeeper.stats.StatsLogger
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 9 more

Reply:
@sijie after rebasing changes and rebuilding, this error is not reproing now, wondering in which commit you fixed it.

Reply:
do you run with a clean build after you switch the branch?

Reply:
probably not..and yeah that could be reason for why it didnt repro without any code change